core/src/test/java/com/sequenceiq/cloudbreak/core/bootstrap/service/container/postgres/PostgresConfigServiceTest.java

```diff
@@ -76,8 +85,9 @@ void setUp() {
     void decorateServicePillarWithPostgresIfNeededTestCertsWhenSslDisabled() {
```

Comment:
Similarly to my comment for `decorateServicePillarWithPostgresIfNeededTestCertsWhenSslEnabled()`, there are two scenarios to test in `decorateServicePillarWithPostgresIfNeededTestCertsWhenSslDisabled()`:
* `stackDto.getCluster().getDbSslRootCertBundle() == null`, where the invocations `dbCertificateProvider.getRelatedSslCerts()` and `clusterService.updateDbSslCert()` will indeed take place, and the the former call returns `new RedbeamsDbSslDetails(Set.of(), false)`.
* `stackDto.getCluster().getDbSslRootCertBundle() == ""` (and also `stackDto.getCluster().getDbSslEnabled == false`). In this case, no calls shall be made to `dbCertificateProvider.getRelatedSslCerts()` and `clusterService.updateDbSslCert()`.

Please extend this test case accordingly with an additional `boolean` controlling whether the `Cluster` is already initialized or not. It is also okay to clone this test case and add a separate one for the other scenario.

Reply:
I suggest either extending this or adding a separate test case for the scenario where `cluster.getDbSslRootCertBundle() == ""`. In this case, no calls shall be made to `dbCertificateProvider.getRelatedSslCerts()` and `clusterService.updateDbSslCert()`.